internal/procfs/procfs_linux.go

```diff
@@ -0,0 +1,105 @@
+package procfs
```

Comment:
does this mean that `docker/insternal/procfs` goes away? Why is the docker one under internal? And why is this under internal?

Reply:
@tonistiigi yes. I don't know why it's under internal. Maybe because this procfs package is not great.

Reply:
can you move it out?

Reply:
yes